aries_cloudagent/config/argparse.py

```diff
@@ -925,6 +925,34 @@ def add_arguments(self, parser: ArgumentParser):
             env_var="ACAPY_AUTO_MEDIATION",
             help="automate all steps of mediation. Default: false.",
         )
+        parser.add_argument(
+            "--mediation-invitation",
+            type=str,
+            dest="mediation_invitation",
+            # metavar="<mediation-invitation>",
+            env_var="ACAPY_MEDIATION_INVITATION",
+            help="Specifies a connection invitation to connect to"
+            "a mediation agent for mediation service. The invitation is provided as"
+            " JSON string e.g. "
+            "{{ @type: did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/connections/1.0/invitation"
+            ", serviceEndpoint: http://192.168.1.13:3005,...",
+        )
+        parser.add_argument(
+            "--auto-mediation-request-on-discovery",
+            action="store_true",
+            env_var="ACAPY_AUTOMATE_MEDIATION_REQUEST_ON_DISCOVERY",
+            help="When enabled, connection completion will trigger a feature discovery"
+            " request over the connection, if mediation is supported a mediation request"
+            " will be sent. Default: false.",
+        )
+        parser.add_argument(
+            "--auto-respond-mediation-grant",
+            action="store_true",
+            env_var="ACAPY_AUTO_RESPOND_MEDIATION_GRANT",
+            help="Automatically respond to mediation grant message with a keylist update"
+            "message containing a newly created did verkey for use as a recipient key"
+            ". Default: false.",
+        )
```

Comment:
These changes probably don't belong in this PR. Any thoughts to the contrary, @burdettadam?

Reply:
Correct, I thought I had already removed some of them.